src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,38 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage
+{
+    public HomePage(WebDriver givenDriver)
+    {
+        super(givenDriver);
+    }
+
+    By userAvatarIcon = By.cssSelector("img.avatar");
+    By allSongs = By.cssSelector("li a.songs");
+    By playBtn = By.cssSelector("[data-testid='play-btn']");
+
+    public WebElement getUserAvatar()
+
+    {
+        return findElement(userAvatarIcon);
+    }
+
+    //Play Song Methods
+    public void chooseAllSongsList()
+    {
+        waitForOverlayDisappear();
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://github.com/testprojp/A47-QA-Automation-Framework/assets/104761482/7a8f3d29-8398-48aa-a46f-76f5e82b7aad)